src/Tasks/Common/Resources/Strings.resx

```diff
@@ -653,4 +653,12 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1137: It is no longer necessary to use the Microsoft.NET.Sdk.WindowsDesktop SDK. Microsoft.NET.Sdk can be used instead.</value>
     <comment>{StrBegin="NETSDK1137: "}</comment>
   </data>
+  <data name="CannotIncludeSymbolsInSingleFile" xml:space="preserve">
+    <value>NETSDK1138: Including symbols in a single file bundle is not supported when publishing for .NET5 or higher.</value>
+    <comment>{StrBegin="NETSDK1138: "}</comment>
+  </data>
+  <data name="CannotIncludeAllContentButNotNativeLibrariesInSingleFile" xml:space="preserve">
+    <value>NETSDK1139: Including all content in a single file bundle also includes native libraries. Either set IncludeNativeLibrariesForSelfExtract to true, or set IncludeAllContentForSelfExtract to false.</value>
```

Comment:
Or unset `IncludeNativeLibrariesForSelfExtract`, right? Maybe just state the contract, something like, "if IncludeAllContentForSelfExtract is true, IncludeNativeLibrariesForSelfExtract must not be false"?

Reply:
Fixed!